### ISSUE TITLE
Release Slicer v4.11.20200930

### ISSUE THREAD

**Issue by jcfr** [Slicer/Slicer#5217]
This is a snapshot release intended to provide user with a "stable" version of Slicer built against VTK 8.2

Checklist Day 1:
* [x] Update Slicer.crt and commit change. See https://github.com/Slicer/Slicer/commit/2607894aaece9ea71f1ec726c4ed7d5f7e8bff4a
* [x] CMakeLists.txt: Update the Slicer version information for the release. See https://github.com/Slicer/Slicer/commit/002be18086c3054bb834dd674401ba624629e4f6
* [x] Tag the repository with `v4.11.20200930`. See [here](https://github.com/Slicer/Slicer/releases/tag/v4.11.20200930). I confirmed packages generated using the corresponding hash can be installed:
  * Linux package :heavy_check_mark: 
  * macOS package :heavy_check_mark: 
  * Windws package :heavy_check_mark: 
* [x] CMakeLists.txt: Update the Slicer version information for the development. See https://github.com/Slicer/Slicer/commit/87eee93d80296fba7fd627ea0602a850acf01a55
* [x] Tag and publish SlicerBuildEnvironment docker image. See [slicer/buildenv-qt5-centos7:slicer-4.11-2020.09.30](https://hub.docker.com/layers/slicer/buildenv-qt5-centos7/slicer-4.11-2020.09.30/images/sha256-ee8da10829b5812bf774f7c2d3d9d39b1d693e62edbc732e258143c6c7bcc5bc?context=explore)
* [x] Update release scripts. See https://github.com/Slicer/DashboardScripts/pull/33
* [x] Disable regular nightly builds
* [x] Update ExtensionsIndex: Create `4.11` branch
* [x] Update CDash: Group `Extensions-4.11-Nightly` created on http://slicer.cdash.org/index.php?project=Slicer4
* [x] Generate application and extension packages
  * Linux: in progress :heavy_check_mark: 
  * macOS: in progress :heavy_check_mark: 
  * Windows: in progress :heavy_check_mark: 
* [x] <s>Create release branch</s> We do not create release branch for snapshot releases
* [x] Update Release Details on the wiki

Checklist Day2:
* [x] Re-enable regular nightly builds
* [ ] Update Slicer wiki: Create `4.11` namespace
* [x] <s>Prepare Slicer Announcement</s>. We do not announce snapshot releases
* [ ] Clean-up older nightly packages
* [x] Manually sign packages
* [x]  Tag release packages
* [x]  Confirm that packages are tagged
* [x] <s>Version NA-MIC data tree</s> We are moving away of midas in favor of release assets. See https://github.com/Slicer/SlicerTestingData#readme and https://github.com/Slicer/SlicerDataStore#readme
* [ ] Update external website.
* [x] Update ExtensionStats extension: ✔️ See https://github.com/Slicer/SlicerDeveloperToolsForExtensions/commit/9d29f8af961797b224926ee42a0ae1bfb1de379e
* [x] <s>Publish Slicer Announcement</s>. We do not announce snapshot releases

References:
* https://www.slicer.org/wiki/Documentation/Nightly/Developers/ReleaseProcess

**Comment by jcfr** [comment]
@sjh26  Would appreciate your help to finalize the signing of the v4.11.20200930 packages, there are available on midas:

See http://slicer.cdash.org/index.php?project=Slicer4&date=2020-09-30&filtercount=1&showfilters=1&field1=groupname&compare1=63&value1=Packages

Direct links:
* Linux: https://slicer.kitware.com/midas3/item/548408
* macOS: https://slicer.kitware.com/midas3/item/548558
* Windows: https://slicer.kitware.com/midas3/item/548506

In a nutshell, the following steps need to be followed:
* https://www.slicer.org/wiki/Documentation/Nightly/Developers/ReleaseProcess#Manually_sign_packages
* https://www.slicer.org/wiki/Documentation/Nightly/Developers/ReleaseProcess#Tag_release_packages

**Comment by sjh26** [comment]
Sure, I will work on this today.

**Comment by sjh26** [comment]
Packages have been signed and uploaded, and are available on download.slicer.org

**Comment by pieper** [comment]
I was able to get the Slicer-4.11.20200930-macosx-amd64 from download.slicer.org but it appears not to have been signed.

![image](https://user-images.githubusercontent.com/126077/94964057-1a3adb80-04c7-11eb-8189-e908203a3f54.png)

**Comment by sjh26** [comment]
Okay, I'm looking into this

**Comment by sjh26** [comment]
@pieper Can you try this one: https://drive.google.com/file/d/1Fy7fvpIac5Xrtu1gkPLaGWNCyefsppMa/view?usp=sharing

**Comment by pieper** [comment]
Oddly enough the same thing.

![image](https://user-images.githubusercontent.com/126077/94968539-1dd26080-04cf-11eb-9758-203276d14cf0.png)

**Comment by jcfr** [comment]
@Sam: i wonder if we now have to notarize the app. This could be done with
the i referenced the other day.

**Comment by jcfr** [comment]
See https://github.com/akeru-inc/xcnotary

**Comment by sjh26** [comment]
Yeah, it must be something this.  If I transfer the file using ftp onto a system its fine.  If I upload then download it, it cannot be verified.

**Comment by sjh26** [comment]
Can I do this on the signing machine?  I'll need to have access to the key, but we don't have brew or anything else set up for this.

**Comment by sjh26** [comment]
I'm not sure that this is notarization though. Example of notarization error: https://www.cdfinder.de/guide/blog/apple_hell.html

**Comment by sjh26** [comment]
Update:  We currently suspect that the failed signing is related to lurking build rpaths.  Investigation continues.

**Comment by cpinter** [comment]
Can we close this?

**Comment by jcfr** [comment]
> * [ ] Update external website.

 After I update wikipedia, I will close this. See https://en.wikipedia.org/wiki/3D_Slicer

cc: @pieper @sjh26 @lassoan @cpinter

**Comment by jamesobutler** [comment]
It appears the Slicer version on the Wikipedia page was updated. Should this issue be closed now?
![image](https://user-images.githubusercontent.com/15837524/103394595-7974b000-4af7-11eb-8728-8c07e1d6fb4f.png)

**Comment by jcfr** [comment]
Indeed, thanks for the reminder.
